pkg/cmd/server/api/validation/master.go

```diff
@@ -184,6 +183,9 @@ func ValidateServiceAccountConfig(config api.ServiceAccountConfig, builtInKubern
 	if !managedNames.Has(bootstrappolicy.BuilderServiceAccountName) {
 		// TODO: warn that default builder service account won't be auto-created
 	}
+	if !managedNames.Has(bootstrappolicy.DeployerServiceAccountName) {
```

Comment:
I added infrastructure for warnings, seems like this ought to actually produce a warning.